src/plugins/cosmos/components/modals/Send/TxFeeRadioGroup.tsx

```diff
@@ -0,0 +1,130 @@
+import { Box, Button, ButtonGroup, Radio, Spinner, useColorModeValue } from '@chakra-ui/react'
```

Comment:
The cosmos `<TxFeeRadioGroup />` is already implemented and has all the cosmos-specific logic, please reuse it.

https://github.com/shapeshift/web/blob/bf2380bfa87ae064cd58a413da0b8354b80f6f0c/src/plugins/cosmos/components/TxFeeRadioGroup/TxFeeRadioGroup.tsx



Also this is just a copy and paste. It doesn't contain Cosmos specific logic or copy. Please remove it.